### DISCUSSION TITLE
Controlling camera states

### DISCUSSION THREAD

**Discussion by jappish84** [mrlt8/docker-wyze-bridge#92] [Ideas]
First of all, great work with wyze-bridge! Love it! 

Would it be possible to use wyze-bridge to control the state of the cameras, i.e. Turning cameras on and off?

I'm now using ifttt automations to do that, but it's suboptimal, especially with the ifttt free account limitations

**Comment by mrlt8** [comment]
Unfortunately, this project is aimed to be more of a minimal set it and forget it type of thing so it doesn't take any interactions or inputs once it's up and running.

I'm always open to pull requests if anyone finds an interesting way to accomplish this though!

**Reply by jappish84** [reply]
Understandable, thanks again.

I actually found this repo that does exactly what I was after, maybe it'll help someone else out 

https://github.com/JoshuaMulliken/ha-wyzeapi
